src/akkudoktoreosserver/flask_server.py

```diff
@@ -26,6 +27,25 @@
 )
 
 
+def isfloat(num: Any) -> bool:
```

Comment:
you could add a TypeGuard:
`
from typing import TypeGuard
def isfloat(num: Any) -> TypeGuard[float]
`

Reply:
oh that's a very nice function I did not know about. 
According to PEP 647 thats in Version 3.10 introduced. Currently we need 3.8+ according to the readme. Would 3.10+ be a problem? Ill add it

Reply:
I did not check the required python version, my bad.

Reply:
do you see any reason not to change the python version needed? 3.10 is already 4 years old.

Reply:
personally I would even go to 11 or even 13. But as the program is also distributed as a package that other people can implement, it is better to support more versions.